config/set/php81.php

```diff
@@ -17,11 +17,11 @@
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
     $services->set(ReturnNeverTypeRector::class);
-    $services->set(MyCLabsClassToEnumRector::class);
+    //$services->set(MyCLabsClassToEnumRector::class);
     $services->set(MyCLabsMethodCallToEnumConstRector::class);
     $services->set(FinalizePublicClassConstantRector::class);
     $services->set(ReadOnlyPropertyRector::class);
-    $services->set(SpatieEnumClassToEnumRector::class);
+    //$services->set(SpatieEnumClassToEnumRector::class);
     $services->set(Php81ResourceReturnToObjectRector::class);
```

Comment:
These should be included in set so others can use them, but be skipped in our `rector.php`.

Reply:
updated https://github.com/rectorphp/rector-src/pull/1374/commits/0ee361c471aa92f06cf2847699ba81693aa2d676